packages/front-b2b-link-rn/src/Front.tsx

```diff
@@ -53,9 +52,12 @@ const FrontFinance = (props: {
     ) {
       setShowWebView(false);
     }
+    if (type === "showClose") {
+      props.onClose && props?.onClose();
```

Comment:
why do we need `props?` here?

Reply:
due to hiding Frontfinance on main application fore.g example app